docs/getting-started/installation.md

```
@@ -68,23 +66,21 @@ specfact init --install all
 specfact init ide --install-deps
-
-# Initialize for specific IDE and install dependencies
 specfact init ide --ide cursor --install-deps
 ```
 
 **Important**: SpecFact CLI does **not** ship with built-in AI. `specfact init ide` installs prompt templates for supported IDEs so your chosen AI copilot can call SpecFact commands in a guided workflow.
 
-### Option 3: Container
+[More options ↓](#more-options)
+
+## More options
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)